keyboards/massdrop/ctrl/keymaps/endgame/keymap.c

```diff
@@ -0,0 +1,428 @@
+        CAPS: 50, A: 51,    S: 52,    D: 53,    F: 54,    G: 55,    H: 56,    J: 57,    K: 58,    L: 59,    SCLN: 60, QUOT: 61, ENT: 62, \
+        LSFT: 63, Z: 64,    X: 65,    C: 66,    V: 67,    B: 68,    N: 69,    M: 70,    COMM: 71, DOT: 72,  SLSH: 73, RSFT: 74,                               UP: 75, \
+        LCTL: 76, LGUI: 77, LALT: 78,                   SPC: 79,                                  RALT: 80, Fn: 81,   APP: 82,  RCTL: 83,           LEFT: 84, DOWN: 85, RGHT: 86 \
+    ),
+	[MATRIX] = LAYOUT(
+	    0,       1,       2,       3,       4,       5,       6,       7,       8,       9,       10,      11,      12,                 13,      14,      15,      \
+	    16,      17,      18,      19,      20,      21,      22,      23,      24,      25,      26,      27,      28,      29,        30,      31,      32,      \
+	    33,      34,      35,      36,      37,      38,      39,      40,      41,      42,      43,      44,      45,      46,        47,      48,      49,      \
+	    50,      51,      52,      53,      54,      55,      56,      57,      58,      59,      60,      61,      62,      \
+	    63,      64,      65,      66,      67,      68,      69,      70,      71,      72,      73,      74,                                   75,      \
+	    76,      77,      78,                        79,                                 80,      81,      82,      83,                 84,      85,      86      \
+	),
+    */
+};
+
+// Runs just one time when the keyboard initializes.
+void matrix_init_user(void) {
+    // Enable or disable debugging
+    debug_enable=true;
+    debug_matrix=true;
+    debug_keyboard=true;
+    debug_mouse=true;
+
+    idle_second_counter = 0;
+    rgb_time_out_seconds = RGB_DEFAULT_TIME_OUT;
+    rgb_time_out_enable = true;
+    rgb_enabled_flag = true;
+    rgb_time_out_user_value = true;
+    rgb_time_out_fast_mode_enabled = false;
+    rgb_time_out_saved_flag = rgb_matrix_get_flags();
+};
+
+// Runs constantly in the background, in a loop.
+void matrix_scan_user(void) {
+    if(rgb_time_out_enable && rgb_enabled_flag) {
+        if (timer_elapsed(idle_timer) > MILLISECONDS_IN_SECOND) {
+            idle_second_counter++;
+            idle_timer = timer_read();
+        }
+
+        if (idle_second_counter >= rgb_time_out_seconds) {
+            rgb_time_out_saved_flag = rgb_matrix_get_flags();
+            rgb_matrix_set_flags(LED_FLAG_NONE);
+            rgb_matrix_disable_noeeprom();
+            rgb_enabled_flag = false;
+            idle_second_counter = 0;
+        }
+    }
+};
+
+#define MODS_SHIFT  (get_mods() & MOD_BIT(KC_LSHIFT) || get_mods() & MOD_BIT(KC_RSHIFT))
+#define MODS_CTRL  (get_mods() & MOD_BIT(KC_LCTL) || get_mods() & MOD_BIT(KC_RCTRL))
+#define MODS_ALT  (get_mods() & MOD_BIT(KC_LALT) || get_mods() & MOD_BIT(KC_RALT))
```

Comment:
These can be replaced with `get_mods() & MOD_MASK_SHIFT` etc.

Reply:
Changed and tested on hardware. Didn't know about MOD_MASK actually. Thank you for the info.